wire-schema/src/main/java/com/squareup/wire/internal/protoparser/EnumConstantElement.java

```diff
@@ -72,7 +86,8 @@ public EnumConstantElement build() {
       checkNotNull(name, "name");
       checkNotNull(tag, "tag");
 
-      return new AutoValue_EnumConstantElement(name, tag, documentation, immutableCopyOf(options));
+      return new AutoValue_EnumConstantElement(name, tag, documentation,
+          ImmutableList.copyOf(options));
```

Comment:
We can put `ImmutableList` as the method return types too now that this is hidden and we have the Guavas.